Tasks/ServiceFabricDeploy/deploy.ps1

```diff
@@ -52,6 +53,12 @@ try {
     # Connect to cluster
     Connect-ServiceFabricClusterFromServiceEndpoint -ClusterConnectionParameters $clusterConnectionParameters -ConnectedServiceEndpoint $connectedServiceEndpoint
 
+    if ($configureDockerSettings)
```

Comment:
Why is this required to be done in Deploy task? Shouldn't it be part of manifest update task? I was assuming that there won't be any change needed in ServiceFabricDeploy task. Am I missing something?

Reply:
Should we rather task SF connection endpoint as input in "ServicefabricUpdateManifest" task itself? This will require just one additional input in that task instaed of 4-5 inputs here